homeassistant/components/xiaomi_miio/const.py

```diff
@@ -235,11 +250,14 @@
 FEATURE_SET_FAN_LEVEL = 4096
 FEATURE_SET_MOTOR_SPEED = 8192
 FEATURE_SET_CLEAN = 16384
-FEATURE_SET_OSCILLATION_ANGLE = 32768
-FEATURE_SET_OSCILLATION_ANGLE_MAX_140 = 65536
-FEATURE_SET_DELAY_OFF_COUNTDOWN = 131072
-FEATURE_SET_LED_BRIGHTNESS_LEVEL = 262144
-FEATURE_SET_FAVORITE_RPM = 524288
+FEATURE_SET_OSCILLATION_ANGLE_MAX_120_STEP_1 = 32768
```

Comment:
I would not split up here all varieties, better to map the min max en step attributes to a class to keep this scalable.

Reply:
I think now is much better. What do you think?

Reply:
Look much better indeed. Nice work